pkg/native/serializers/serializer_cdx.go

```diff
@@ -252,7 +296,7 @@ func (s *SerializerCDX) nodeToComponent(n *sbom.Node) *cdx.Component {
 	if n.Hashes != nil && len(n.Hashes) > 0 {
 		c.Hashes = &[]cdx.Hash{}
 		for algoString, hash := range n.Hashes {
-			if algoVal, ok := sbom.HashAlgorithm_value[algoString]; ok {
+			if algoVal, ok := sbom.HashAlgorithm_value[strings.ToUpper(algoString)]; ok {
```

Comment:
OK, this could solve some cases but [cdx uses dashes in the algorithm labels](https://github.com/CycloneDX/specification/blob/cc15c8550e87dfa9ee051e6795e74a3c6c641264/schema/bom-1.5.schema.json#L705-L719) so, if the value comes from an [SPDX checksum](https://spdx.github.io/spdx-spec/v2.3/package-information/#710-package-checksum-field), they will not be valid.

Let's get it in for now, it will be solved once we resolve #89

Reply:
Yeah, #89 is definitely the better way.  This is a hack